python_on_whales/exceptions.py

```diff
@@ -9,20 +9,23 @@ def __init__(
         stdout: Optional[bytes] = None,
         stderr: Optional[bytes] = None,
     ):
-        command_launched_str = " ".join(command_launched)
+        self.docker_command = " ".join(command_launched)
+        self.return_code = return_code
+        self.stderr = stderr.decode() if stderr else ""
+        self.stdout = stdout.decode() if stdout else ""
         error_msg = (
-            f"The docker command executed was `{command_launched_str}`.\n"
+            f"The docker command executed was `{self.docker_command}`.\n"
```

Comment:
Please add a small unit test where you inspect the exception and the attributes

Reply:
Done in last version or PR (c810682).